src/Ether.Vsts/Handlers/Queries/GetFirstOrganizationHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Ether.Contracts.Interfaces;
+using Ether.Contracts.Interfaces.CQS;
+using Ether.ViewModels;
+using Ether.Vsts.Dto;
+using Ether.Vsts.Queries;
+
+namespace Ether.Vsts.Handlers.Queries
+{
+    public class GetFirstOrganizationHandler : IQueryHandler<GetFirstOrganization, OrganizationViewModel>
```

Comment:
What's the scenario when you need first org?

Reply:
Discussed offline, #56 created.